exasol/slc/models/language_definition_model.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import Annotated, List
+
+from pydantic import BaseModel, Field
+
+from exasol.slc.models.language_definition_common import (
+    SLCLanguage,
+    SLCParameter,
+    UdfClientRelativePath,
+)
+
+
+class LanguageDefinition(BaseModel):
+    """
+    Contains information about a supported language and the respective path of the UDF client of an Script-Languages-Container.
+    """
+
+    protocol: str
+    aliases: List[str]
+    language: SLCLanguage
+    parameters: List[SLCParameter]
```

Comment:
should we add environment variables, for example for the option parser

Reply:
Hm, let's do it if we decided to really use it, because the alternative would be to use the `parameters` field